sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -11,6 +11,12 @@ public class CommunicationTestEnvironment : TestEnvironment
         public const string ConnectionStringEnvironmentVariableName = "COMMUNICATION_CONNECTION_STRING";
         public const string LiveTestConnectionStringEnvironmentVariableName = "AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING";
         public const string AzurePhoneNumber = "AZURE_PHONE_NUMBER";
+        private const string TestPackagesEnabledEnvironmentVariableName = "TEST_PACKAGES_ENABLED";
+        private const string TestPackagesEnabledDefaultValue = "all";
+        private const string SmsTestPackagesEnabled = "sms";
```

Comment:
Should these variables be here or in the Environment of the specific packages? Each package should have a class extending this class already that would be used for their environment

Reply:
I did it here because I set the access level of ```TEST_PACKAGES_ENABLED``` as private, but putting these prams in their own env could be better

Reply:
We could have a method here that would receive the package name and would say if the tests are enabled or not for that package. Then each extension of this class can use it to see if their tests should run